code/modules/mob/inventory.dm

```diff
@@ -176,8 +176,11 @@ var/list/slot_equipment_priority = list( \
 
 //Drops the item in our active hand. TODO: rename this to drop_active_hand or something
 /mob/proc/drop_item(var/atom/Target)
-	if(hand)	return drop_l_hand(Target)
-	else		return drop_r_hand(Target)
+	if(!Target && !l_hand && !r_hand)
+		var/obj/item/grab/grab = locate() in contents
+		if(grab)
+			return drop_from_inventory(grab)
```

Comment:
This is for simple mobs? Why check `!target`? Also, isn't there a getter now?

Reply:
Checking `Target` to see if something else has called it to drop an equipped item, like the hat on dionaea. Root proc uses similar logic.

Reply:
Not sure what you mean by getter, what would the getter be getting?

Reply:
Meant `get_active_grabs`.

Reply:
Done.